### ISSUE TITLE
Div element missing when running Cypress test in Test Runner

### ISSUE THREAD

**Issue by ATarkovski** [cypress-io/cypress#25916]
### Current behavior

In the webpage I'm testing (https://www.izotope.com/), there is a mega menu that it's shown when hovering over the Products option.

<img width="866" alt="Screenshot 2023-02-22 at 15 17 39" src="https://user-images.githubusercontent.com/35202071/220647910-f5e472ca-329f-4cd5-84f2-e690ef461814.png">

In my test I was trying to get the menu displayed with cy.trigger() (see https://docs.cypress.io/api/commands/hover) and nothing happened.

Then, I took a look to the DOM in Cypress and I saw that the problem is that the whole div section is missing. 

I have tried with different versions of Cypress, current (12.6) and pre v10 (9.5.1) and launching different browsers, Electron, Chrome, Edge, and the div element is missing everywhere.

<img width="1368" alt="Screenshot 2023-02-22 at 14 52 07" src="https://user-images.githubusercontent.com/35202071/220640672-38cd8388-1d16-448a-8785-57af9e20d5c2.png">

### Desired behavior

I would like to see this particular div section in Cypress in order to validate the mega menu is displayed.

<img width="1031" alt="Screenshot 2023-02-22 at 14 50 52" src="https://user-images.githubusercontent.com/35202071/220641126-9080cb83-2f92-4e62-88d7-f6a12c3492e7.png">

### Test code to reproduce

cy.visit('https://www.izotope.com')

### Cypress Version

12.6.0 and 9.5.1

### Node version

19.1.0

### Operating System

macOS 13.2

**Comment by flotwig** [comment]
@ATarkovski This seems like it could possibly be a bug in your site related to viewport size.

I can reproduce what you're saying. But if I change the test to render in a 1920x1080 viewport instead of the 1000x660 default, I am able to hover over `Products`:
![image](https://user-images.githubusercontent.com/1151760/221951706-8c963d6f-c228-4911-9080-75b99713d56b.png)

Is there a chance that your `<div>` is conditionally rendered only on initial viewport size?

**Comment by ATarkovski** [comment]
Hi @flotwig. Thanks for your answer. I tried with the viewport you suggested and the mega menu is appearing in the DOM and I'm able to hover over it. I switched back to the initial viewport I had set, 1280x720px and now it works. The mega menu is disabled for viewports less than 1024px wide, so it make sense that it failed if I had set a viewport like that. Weird.

Anyway, thanks for your support! I think we are good closing the ticket.
